docs/process/release.md

```diff
@@ -138,19 +141,21 @@ After closing the tab, you have successfully logged in, it is time to publish.
 
 **It is a good idea to wait for CI to be green on the patch branch and tag before doing the release.**
 
-Check out the patch tag (e.g. `v6.7.8`), then run:
-```sh
-devkit-admin publish
+Check out the patch branch (e.g. `9.1.x`), then run:
+```bash
+yarn # Reload dependencies
+./bin/devkit-admin publish
```

Comment:
The `bin` here is without the leading `.`. 
Also, maybe better use;
`yarn admin publish`

Reply:
I switched it to `yarn admin`, except for changelog generation, because we need to copy the output which will include:

```
yarn run v1.22.0
$ node ./bin/devkit-admin
```

If we use `yarn admin` rather than calling `devkit-admin` directly.

Is `devkit-admin` supposed to be installed on the path? I always just run it locally within the repo. Is there a different way I'm supposed to use this?

Reply:
I personally always use `yarn admin`. I don't have `devkit-admin` installed in my path and always try to keep my paths as clean as possible.

Reply:
Keen pointed out that we can use `yarn run -s admin` to silence the extra text. Updated to use that format.